games-strategy/7kaa/7kaa-2.15.4p1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="Seven Kingdoms: Ancient Adversaries"
+DESCRIPTION="Seven Kingdoms, designed by Trevor Chan, brings a unique blend of Real-Time Strategy\
+with the addition of trade, diplomacy, and espionage.\
+\
```

Comment:
Line 2 space before the backslash, line 3 and 4, no backslashes (looks better to start line 5 on a new line), you could even leave out line 4.
```diff
-DESCRIPTION="Seven Kingdoms, designed by Trevor Chan, brings a unique blend of Real-Time Strategy\
-with the addition of trade, diplomacy, and espionage.\
-\
+DESCRIPTION="Seven Kingdoms, designed by Trevor Chan, brings a unique blend of Real-Time Strategy \
+with the addition of trade, diplomacy, and espionage.
```